### ISSUE TITLE
iOS LaunchScreen duration time

### ISSUE THREAD

**Issue by ptysbilbo** [Gamua/Adobe-Runtime-Support#416]
Hi,
I noticed, that the time the launch screen is shown on iOS is significantly longer with the newest Air SDK 33.1.1.190.
(I'm using "LaunchScreen.storyboardc" and "Assets.car")

When I compile my app with 33.1.1.98, the image is displayed for no more than 0,5 s and the app is fully loaded.
With 33.1.1.190 it takes about 2 secs or even more.

No changes are made to the app, so it's not more complex and therefore needs more time to load.
Everything is completely the same, only different SDK is being used.

I'm using VS code to compile and testing on iPad Air 2 with iOS 12.4.1.

Is there any way to shorten this time?

Thanks

**Comment by 2cde** [comment]
+1

**Comment by KaseYu** [comment]
+1

**Comment by ajwfrost** [comment]
Yes the changes here are where we added the launch storyboard I believe, so the mechanism has changed. The aim was to get more of a parity in behaviour then between iOS and Android, it's a little complex as we're trying to load in the runtime and initial SWF file in the background whilst this is going on. I'll check to see whether we can do anything here..

thanks
